index.md

```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a declarative domain specific grammar and syntax that allows you to **annotate** and **validate** JSON documents. JSON Schema enables the confident and reliable use of the JSON data format.
```

Comment:
Isn't "grammar and syntax" just a more verbose way to say "language"? Also, domain specific language (DSL) is specific well defined technical term. Saying "language" in a different way shouldn't change the meaning, but it does increase the cognitive load of parsing this sentence. Of course it could just be that the sentence is easy to parse because I'm familiar with these technical terms. Not everyone is.

Reply:
JSON Schema is way closer to a language than a grammar or syntax. Our grammar and syntax is entirely imported, and in no way domain-specific.

> Not everyone is.

Most of our intended audience should be familiar with the terms, but our front page should be written for the widest possible audience, not just our intended audience.

Reply:
Let's go with language then!